environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -12,15 +14,16 @@ import (
 	"github.com/fsouza/go-dockerclient"
 	"github.com/Shopify/sarama"
 	"encoding/json"
-	//"github.com/minio/minio-go"
-	"github.com/benchflow/commons/minio"
+	//"github.com/benchflow/commons/minio"
 	"strconv"
 )
 
 type Container struct {
+	Name         string
 	ID           string
 	statsChannel chan *docker.Stats
 	doneChannel chan bool
+	IsHost       bool
```

Comment:
Use `Network`: `host`/`bridge`....Store the actual network, then you decide in the code what to do.